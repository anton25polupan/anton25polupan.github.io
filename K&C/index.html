<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="scss/main.css">
    <link rel="stylesheet" href="scss/style_mobile.css">
    <title>Document</title>
</head>
<body>
    <div class="container">
        <aside class="sidebar">
            <ul class="sidebar__lists">
                <li class="sidebar__list-item">
                    <img src="img/user.png" alt="">
                </li>
                <li class="sidebar__list-item active">
                    <a href="#">Сообщения<span>3</span></a>
                </li>
                <li class="sidebar__list-item">
                    <ul>
                        <li>
                            <a href="#">Профиль</a>
                        </li>
                        <li>
                            <a href="#">Настройки</a>
                        </li>
                    </ul>
                </li>
                <li class="sidebar__list-item">
                    <ul>
                        <li>
                            <a href="#">Вопросы</a>
                        </li>
                        <li>
                            <a href="#">Написать модератору</a>
                        </li>
                    </ul>
                </li>
                <li class="sidebar__list-item">
                    <a href="#">Купить подписку</a>
                </li>
            </ul>    
        </aside>
        <div class="content__desktop">
            <main class="content">
                <ul class="content__control-top">
                    <li>Удалить</li>
                    <li>Добавить архив</li>
                    <li>Непрочитанные</li>
                    <li>Архив</li>
                    <form>
                        <input placeholder="Искать">
                        <span></span>
                    </form>
                </ul>
                <div class="content__messages">
                    <table>
                        <tbody>
                            <tr> 
                                <td>
                                    <input type="checkbox" id="inp_check1">
                                    <label for="inp_check1"></label>
                                </td>
                                <td></td>
                                <td>Александр Анто…</td>
                                <td class="shop"></td>
                                <td>здравствуйте. как долго придется ждать нож кото</td>
                                <td>13/04/2018</td>
                                <td>14:02</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check2">
                                    <label for="inp_check2"></label>
                                </td>
                                <td class="online"></td>
                                <td>Gun Machine</td>
                                <td></td>
                                <td>а почему на ноже нет ни одной царапины?</td>
                                <td>01/12/2018</td>
                                <td>00:14</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check3">
                                    <label for="inp_check3"></label>
                                </td>
                                <td></td>
                                <td>Диана Павлова</td>
                                <td></td>
                                <td>нет, мне не нужно.если и вам тоже пофиг, то выб</td>
                                <td>13/04/2018</td>
                                <td>14:02</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check4">
                                    <label for="inp_check4"></label>
                                </td>
                                <td class="online"></td>
                                <td>Otto Zweig</a></td>
                                <td></td>
                                <td>thank you! that's great!</td>
                                <td>01/12/2018</td>
                                <td>00:14</td>
                            </tr>
                            <tr> 
                                <td>
                                    <input type="checkbox" id="inp_check5">
                                    <label for="inp_check5"></label>
                                </td>
                                <td></td>
                                <td>Александр Анто…</td>
                                <td class="shop"></td>
                                <td>здравствуйте. как долго придется ждать нож кото</td>
                                <td>13/04/2018</td>
                                <td>14:02</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check6">
                                    <label for="inp_check6"></label>
                                </td>
                                <td class="online"></td>
                                <td>Gun Machine</td>
                                <td></td>
                                <td>а почему на ноже нет ни одной царапины?</td>
                                <td>01/12/2018</td>
                                <td>00:14</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check7">
                                    <label for="inp_check7"></label>
                                </td>
                                <td></td>
                                <td>Диана Павлова</td>
                                <td></td>
                                <td>нет, мне не нужно.если и вам тоже пофиг, то выб</td>
                                <td>13/04/2018</td>
                                <td>14:02</td>
                            </tr>
                            <tr>
                                <td>
                                    <input type="checkbox" id="inp_check">
                                    <label for="inp_check"></label>
                                </td>
                                <td class="online"></td>
                                <td>Otto Zweig</a></td>
                                <td></td>
                                <td>thank you! that's great!</td>
                                <td>01/12/2018</td>
                                <td>00:14</td>
                            </tr>
                        </tbody>
                    </table>
                </div>
            </main>
            <div class="content__control-bottom">
                <ul>
                    <li><a href="#">1</a></li>
                    <li class="focus"><a href="#">2</a></li>
                    <li><a href="#">3</a></li>
                    <li><a href="#">4</a></li>
                    <li><a href="#">5</a></li>
                    <li><a href="#">6</a></li>
                    <li><a href="#">7</a></li>
                    <li><a href="#">8</a></li>
                    <li><a href="#">9</a></li>
                    <li><a href="#">10</a></li>
                    <li><a href="#">...</a></li>
                    <li><a href="#">123</a></li>
                    <a class='next_arrow' href="#" role="button"></a>
                </ul>
            </div>
        </div>
        <main class="content__mobile">
            <div class="content__control-top">
                <div class="sidebar__btn">
                    <span class="menu__line"></span>
                    <span class="menu__line"></span>
                    <span class="menu__line"></span>
                </div>
                <form>
                    <input placeholder="Искать">
                    <span></span>
                </form>
                <ul class="control__list">
                    <li>Удалить</li>
                    <li>Добавить архив</li>
                    <li>Непрочитанные</li>
                    <li>Архив</li>
                </ul>
            </div>
            <div class="content__messages_mobile">
                <div class="messages_mobile-list">
                    <ul> 
                        <li></li>
                        <li>Александр Анто…</li>
                        <li class="shop"></li>
                        <li>здравствуйте. как долго придется ждать нож кото</li>
                        <li>13/04/2018</li>
                        <li>14:02</li>
                    </ul>
                    <ul>
                        <li class="online"></li>
                        <li>Gun Machine</li>
                        <li></li>
                        <li>а почему на ноже нет ни одной царапины?</li>
                        <li>01/12/2018</li>
                        <li>00:14</li>
                    </ul>
                    <ul>
                        <li></li>
                        <li>Диана Павлова</li>
                        <li></li>
                        <li>нет, мне не нужно.если и вам тоже пофиг, то выб</li>
                        <li>13/04/2018</li>
                        <li>14:02</li>
                    </ul>
                    <ul>
                        <li class="online"></li>
                        <li>Otto Zweig</li>
                        <li></li>
                        <li>thank you! that's great!</li>
                        <li>01/12/2018</li>
                        <li>00:14</li>
                    </ul>
                    <ul> 
                        <li></li>
                        <li>Александр Анто…</li>
                        <li class="shop"></li>
                        <li>здравствуйте. как долго придется ждать нож кото</li>
                        <li>13/04/2018</li>
                        <li>14:02</li>
                    </ul>
                    <ul>
                        <li class="online"></li>
                        <li>Gun Machine</li>
                        <li></li>
                        <li>а почему на ноже нет ни одной царапины?</li>
                        <li>01/12/2018</li>
                        <li>00:14</li>
                    </ul>
                    <ul>
                        <li></li>
                        <li>Диана Павлова</li>
                        <li></li>
                        <li>нет, мне не нужно.если и вам тоже пофиг, то выб</li>
                        <li>13/04/2018</li>
                        <li>14:02</li>
                    </ul>
                    <ul>
                        <li class="online"></li>
                        <li>Otto Zweig</li>
                        <li></li>
                        <li>thank you! that's great!</li>
                        <li>01/12/2018</li>
                        <li>00:14</li>
                    </ul>
                </div>
            </div>
            <div class="content__control-bottom">
                <ul>
                    <li><a href="#">1</a></li>
                    <li class="focus"><a href="#">2</a></li>
                    <li><a href="#">3</a></li>
                    <li><a href="#">4</a></li>
                    <li><a href="#">5</a></li>
                    <li><a href="#">6</a></li>
                    <li><a href="#">7</a></li>
                    <li><a href="#">8</a></li>
                    <li><a href="#">9</a></li>
                    <li><a href="#">10</a></li>
                    <li><a href="#">...</a></li>
                    <li><a href="#">123</a></li>
                    <a class='next_arrow' href="#" role="button"></a>
                </ul>
            </div>    
        </main>
    </div>
    <script>
        let sidebarBtn = document.querySelector('.sidebar__btn');
        let sidebarShow = document.querySelector('.sidebar');
        sidebarBtn.addEventListener('click', function(){
            sidebarBtn.classList.toggle('show');
            sidebarShow.classList.toggle('show');
        })
      </script>     
</body>
</html>