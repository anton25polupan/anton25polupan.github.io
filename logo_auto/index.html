<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Logo Auto</title>
    <link rel="stylesheet" href="sass/style.css">
    <link href="https://fonts.googleapis.com/css?family=Rajdhani:300,regular,500,600,700" rel="stylesheet" />
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,regular,italic,600,600italic,700,700italic,800,800italic" rel="stylesheet" />
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
    <script src="js/owl.carousel.js"></script>
    <link rel="stylesheet" href="sass/owl.carousel.min.css">
    <link rel="stylesheet" href="sass/owl.theme.default.min.css">
</head>
<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <h1 class="logo"><a href="#">Logo<span>auto</span></a></h1>
                <nav class="nav">
                    <ul class="nav__items">
                        <a href="#"><li class="nav__item">Latest Competitions</li></a>
                        <a href="#"><li class="nav__item">How it Works</li></a>
                        <a href="#"><li class="nav__item">Gift Cards</li></a>
                        <a href="#"><li class="nav__item">Winners</li></a>
                        <a href="#"><li class="nav__item">Entry Lists</li></a>
                        <a href="#"><li class="nav__item">Live Draws</li></a>
                    </ul>
                </nav>
                <div class="nav__btn">
                    <span class="menu__line"></span>
                    <span class="menu__line"></span>
                    <span class="menu__line"></span>
                </div>
                <div class="brake"></div>
                <div class="user">
                    <a href="#" class="user__log">Login</a>
                    <button class="btn__registr">Register</button>
                </div>
            </div>
            <div class="header__intro">
                <div class="intro__inner-info">
                <h2 class="intro__subtitle">Could you be our
                    <span>next winner?</span>
                    </h2>
                <button class="btn__enter" href="#">Enter now</button>
                </div>
                <div class="intro__inner-img">
                    <img src="img/header__inner/car.png" alt="car">
                </div>
            </div>      
        </div>
    </header>
    <section class="section-competition">
        <div class="container">
            <h2 class="section__title"> Latest Competitions</h2>
            <div class="section__cards">
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_competition/audi_2020.png" alt="Audi S5 Coupe">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">2020 Audi S5 Coupe</h3>
                        <p class="cost">$19.99 <span>$54.68</span></p>
                        <p class="tickets">5878 tickets remaining</p>
                    </div>
                    <a href="#" class="card__link"></a>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_competition/ford_2016.png" alt="2016 Ford Mustang GT">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">2016 Ford Mustang GT</h3>
                        <p class="cost">$7.99 <span>$9.68</span></p>
                        <p class="tickets">3290 tickets remaining</p>
                    </div>
                    <a href="#" class="card__link"></a>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_competition/bugatti_2020.png" alt="2020 Bugatti Chiron">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">2020 Bugatti Chiron</h3>
                        <p class="cost">$0.99 <span>$1.39</span></p>
                        <p class="tickets">4014 tickets remaining</p>
                    </div>
                    <a href="#" class="card__link"></a>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_competition/porshe_2019.png" alt="2019 Porsche 911 Carrera">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">2019 Porsche 911 Carrera</h3>
                        <p class="cost">$1.99 <span>$2.99</span></p>
                        <p class="tickets">484 tickets remaining</p>
                    </div>
                    <a href="#" class="card__link"></a>
                </div>
            </div>
        </div>
    </section>
    <section class="section-work">
        <div class="container">
            <h2 class="section__title">How it works</h2>
            <div class="section__items">
                <div class="section__item">
                    <div class="item__icon">
                        <img src="img/section_works/touchscreen 1.svg" alt="">
                    </div>
                    <p class="item__info">Choose a competition and how many entries you would like</p>
                </div>
                <div class="section__item">
                    <div class="item__icon">
                        <img src="img/section_works/online-payment 1.svg" alt="">
                    </div>
                    <p class="item__info">Answer the qualifying question <b>(be sure to answer correctly!)</b> and complete payment</p>
                </div>
                <div class="section__item">
                    <div class="item__icon">
                        <img src="img/section_works/movie-tickets 1.svg" alt="">
                    </div>
                    <p class="item__info">Tickets for all correct answers are entered into the draw, streamed LIVE on Facebook using Googleâ€™s random number generator</p>
                </div>
                <div class="section__item">
                    <div class="item__icon">
                        <img src="img/section_works/winner 1.svg" alt="">
                    </div>
                    <p class="item__info">If you win, we may try to contact you whilst live on Facebook, so be prepared!</p>
                </div>
            </div>
        </div>
    </section>
    <section class="section-reviews">
        <div class="owl-carousel owl-theme">  
            <div class="reviews__body reviews__person-1">
                <div class="reviews__info">
                    <p>"Unbelievable this is a dream come true,</p>
                    <p>no way would I ever think I would own</p>
                    <p>a car like this. What great blokes.</p>
                    <p>Thank you very much!</p>
                    <p>5 star from me!!!"</p>
                    <p class="reviews__name">Danny Churchman</p>
                </div>
                <img class="reviews__img"src="img/section_reviews/man_and_car.png" alt="winner">
            </div>
            <div class="reviews__body reviews__person-2">
                <div class="reviews__info">
                    <p>"Unbelievable this is a dream come true,</p>
                    <p>no way would I ever think I would own</p>
                    <p>a car like this. What great blokes.</p>
                    <p>Thank you very much!</p>
                    <p>5 star from me!!!"</p>
                    <p class="reviews__name">Danny Churchman</p>
                </div>
                <img class="reviews__img"src="img/section_reviews/man_and_car.png" alt="winner">
            </div>
            <div class="reviews__body reviews__person-3">
                <div class="reviews__info">
                    <p>"Unbelievable this is a dream come true,</p>
                    <p>no way would I ever think I would own</p>
                    <p>a car like this. What great blokes.</p>
                    <p>Thank you very much!</p>
                    <p>5 star from me!!!"</p>
                    <p class="reviews__name">Danny Churchman</p>
                </div>
                <img class="reviews__img"src="img/section_reviews/man_and_car.png" alt="winner">
            </div>
        </div>       
    </section>
    <section class="section-winners">
        <div class="container">
            <h2 class="section__title">Latest Winners</h2>
            <div class="section__cards">
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_winners/girl_and_car.png" alt="Audi S5 Coupe">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">Ferrari 458</h3>
                        <p class="data">12 november 2020</p>
                    </div>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_winners/man_and_silver_car.png" alt="Audi S5 Coupe">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">Aquila Gullwing</h3>
                        <p class="data">9 november 2020</p>
                    </div>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_winners/man_and_white_car.png" alt="Audi S5 Coupe">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">2020 Audi S5 Coupe</h3>
                        <p class="data">5 november 2020</p>
                    </div>
                </div>
                <div class="card__item">
                    <div class="item__img">
                        <img src="img/section_winners/man_and_black_car.png" alt="Audi S5 Coupe">
                    </div>
                    <div class="item__info">
                        <h3 class="card__title">Ferrari Purosangue SUV</h3>
                        <p class="data">25 September 2020</p>
                    </div>
                </div>
            </div>    
        </div>
    </section>
    <footer class="footer">
        <div class="container">
            <div class="footer__inner">
                <h1 class="logo">Logo<span>auto</span></h1>
                <nav class="nav">
                    <ul class="nav__items">
                        <a href="#"><li class="nav__item">Latest Competitions</li></a>
                        <a href="#"><li class="nav__item">How it Works</li></a>
                        <a href="#"><li class="nav__item">Gift Cards</li></a>
                        <a href="#"><li class="nav__item">Winners</li></a>
                        <a href="#"><li class="nav__item">Entry Lists</li></a>
                        <a href="#"><li class="nav__item">Live Draws</li></a>
                    </ul>
                </nav>
            </div> 
    </footer>
    <script>
        $('.owl-carousel').owlCarousel({
            loop:true,
            autoplay:true,
            autoplayTimeout:4000, 
            dots:true,
            margin:10,
            nav:false, 
            responsive:{ 
                0:{
                    items:1
    
                },
                600:{
                    items:1
                },
                1000:{
                    items:1
                }
            }
        });
        let menuBtn = document.querySelector('.nav__btn');
        let menu = document.querySelector('.nav');
        menuBtn.addEventListener('click', function(){
        menuBtn.classList.toggle('active');
        menu.classList.toggle('active');
        })
      </script> 
</body>
</html>