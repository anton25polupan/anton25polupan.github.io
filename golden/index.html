<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/style.css">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
    <title>Document</title>
</head>
<body>
  <header id="home" class="header">
    <div class="container">
      <div class="header__inner">
        <a class="logo" href="">Golden</a> 
        <nav class="nav">
          <ul class="nav__items">
            <li class="nav__item"><a class="nav__link "href="#home">Home</a></li>
            <li class="nav__item"><a class="nav__link "href="#serv">Services</a></li>
            <li class="nav__item"><a class="nav__link "href="#portf">Portfolio</a></li>
            <li class="nav__item"><a class="nav__link "href="#about">About</a></li>
            <li class="nav__item"><a class="nav__link "href="#cont">Contact</a></li>
          </ul>
        </nav>
        <div class="nav__btn">
          <span class="menu__line"></span>
          <span class="menu__line"></span>
          <span class="menu__line"></span>
        </div>
      </div>
      <div class="header__intro">
        <div class="intro__inner">
          <h2 class="intro__subtitle">Welcome To Our Studio</h2>
          <h1 class="intro__title">It's nice to meet you</h1>
          <a class="btn"href="#">Tell me more</a>
        </div>
      </div>
    </div>
  </header>
  <section id="serv"class="section--services">
    <div class="container">
      <div class="section__header">
        <h1 class="section__title">Services</h1>
        <h2 class="section__subtitle">Proin iaculis purus consequat sem cure. </h2>
      </div>
      <div class="services">
        <div class="services__items">
          <div class="services__item">
            <div class="services__img">
              <i class="fas fa-shopping-basket"></i>
            </div>
            <h2 class="services__title">E-Commerce</h2>
            <p class="services__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
          <div class="services__item">
            <div class="services__img">
              <i class="fas fa-laptop-medical"></i>
            </div>
            <h2 class="services__title">Responsive Web</h2>
            <p class="services__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
          <div class="services__item">
            <div class="services__img">
              <i class="fas fa-lock"></i>
            </div>
            <h2 class="services__title">Web Security</h2>
            <p class="services__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
        </div>
      </div>  
    </div>
  </section>
  <section id="portf"class="section--portfolio">
    <div class="container">
      <div class="section__header">
        <h1 class="section__title">Our Portfolio</h1>
        <h2 class="section__subtitle">Proin iaculis purus consequat sem cure. </h2>
      </div>
      <div class="portfolio">
        <div class="portfolio__items">
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Ebony&Ivory.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Ebony&Ivory</h3>
                <p class="portfolio__text">Branding</p>
              </div>
            </div>
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Smart.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Smart Stationary</h3>
                <p class="portfolio__text">Print Design</p>
              </div>
            </div>
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Clever_Poster.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Clever Poster</h3>
                <p class="portfolio__text">Print Design</p>
              </div>
            </div>
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Vinyl.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Vinyl Record</h3>
                <p class="portfolio__text">Product Mock-Up</p>
              </div>
            </div>
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Treehouse.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Treehouse Template</h3>
                <p class="portfolio__text">Web Design</p>
              </div>
            </div>
            <div class="portfolio__item">
              <div class="portfolio__img">
                <img src="img/Burned.png" alt="">
              </div>
              <div class="portfolio__content">
                <h3 class="portfolio__title">Burned Logo</h3>
                <p class="portfolio__text">Branding</p>
              </div>
            </div>
        </div>    
      </div>
    </div>
  </section>
  <section id="about"class="section--about">
    <div class="container">
      <div class="section__header">
        <h1 class="section__title">About Us</h1>
        <h2 class="section__subtitle">Proin iaculis purus consequat sem cure. </h2>
      </div>
      <div class="about__items">
        <div class="about__item">
          <div class="about__content">
            <p class="about__date">July 2010</p>
            <p class="about__title">Our Humble Beginnings</p>
            <p class="about__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit.</p>
          </div>
          <div class="about__img">
            <img src="img/about_1.png" alt="">
          </div>
        </div>
        <div class="about__item">
          <div class="about__img">
            <img src="img/about_2.png" alt="">
          </div>
          <div class="about__content">
            <p class="about__date">January 2011</p>
            <p class="about__title">Facing Startup Battles</p>
            <p class="about__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit aenean 
              rhoncus posuere odio in tincidunt.</p>
          </div>
        </div>
        <div class="about__item">
          <div class="about__content">
            <p class="about__date">December 2012</p>
            <p class="about__title">Enter The Dark Days</p>
            <p class="about__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni.</p>
          </div>
          <div class="about__img">
            <img src="img/about_3.png" alt="">
          </div>
        </div>
        <div class="about__item">
          <div class="about__img">
            <img src="img/about_4.png" alt="">
          </div>
          <div class="about__content">
            <p class="about__date">February 2014</p>
            <p class="about__title">Our Trumph</p>
            <p class="about__text">Proin iaculis purus consequat sem cure 
              digni ssim. Donec porttitora entum suscipit 
              aenean rhoncus posuere odio in tincidunt. Proin 
              iaculis purus consequat sem cure digni 
              ssim. Donec porttitora entum suscipit aenean.</p>
          </div>
        </div>
        <div class="about__item">
          <div class="about__img">
            <p>Our<br> Story<br>Continues<br>...</p> 
          </div>
        </div>
      </div>
    </div>
  </section>
  <section class="section--team">
    <div class="container">
      <div class="section__header">
        <h1 class="section__title">Our Amazing Team</h1>
        <h2 class="section__subtitle">Proin iaculis purus consequat sem cure. </h2>
      </div>
      <div class="team__items">
        <div class="team__item">
          <div class="team__img">
            <img src="img/team/1.png" alt="">
          </div>
          <div class="team__content">
            <p class="team__title">Kimberly Thompson</p>
            <p class="team__text">Marketer</p>
            <div class="team__networks">
              <a href="#" class="team__link link__twitter"><i class="fab fa-twitter"></i></a>
              <a href="#" class="team__link link__facebook"><i class="fab fa-facebook-f"></i></a>
              <a href="#" class="team__link link__pinterest"><i class="fab fa-pinterest-p"></i></a>
            </div>
          </div>
        </div>
        <div class="team__item">
          <div class="team__img">
            <img src="img/team/2.png" alt="">
          </div>
          <div class="team__content">
            <p class="team__title">Rico Massimo</p>
            <p class="team__text">Coder</p>
            <div class="team__networks">
              <a href="#" class="team__link link__twitter"><i class="fab fa-twitter"></i></a>
              <a href="#" class="team__link link__facebook"><i class="fab fa-facebook-f"></i></a>
              <a href="#" class="team__link link__google"><i class="fab fa-google-plus-g"></i></a>
            </div>
          </div>
        </div>
        <div class="team__item">
          <div class="team__img">
            <img src="img/team/3.png" alt="">
          </div>
          <div class="team__content">
            <p class="team__title">Uku Masson</p>
            <p class="team__text">Graphic Designer</p>
            <div class="team__networks">
              <a href="#" class="team__link link__facebook"><i class="fab fa-facebook-f"></i></a>
              <a href="#" class="team__link link__pinterest"><i class="fab fa-pinterest-p"></i></a>
            </div>
          </div>
        </div>
      </div>
      <p class="team__info">Proin iaculis purus consequat sem cure 
        digni ssim. Donec porttitora entum suscipit 
        aenean rhoncus posuere odio in tincidunt. Proin 
        iaculis purus consequat sem cure digni 
        ssim. Donec porttitora entum suscipit.</p>
    </div>
  </section>
  <section class="section--logos">
    <div class="container">
      <div class="logos__items">
        <a href="#"><img src="img/logos/Envato.png" alt=""></a>
        <a href="#"><img src="img/logos/WordPress.png" alt=""></a>
        <a href="#"><img src="img/logos/Tuts+.png" alt=""></a>
        <a href="#"><img src="img/logos/Microlancer-copy.png" alt=""></a>
    </div>   
  </section>
  <section id="cont"class="section--contacts">
    <div class="container">
      <div class="section__header">
        <h1 class="section__title">Contact Us</h1>
        <h2 class="section__subtitle">Proin iaculis purus consequat sem cure. </h2>
      </div>
      <form class="contacts__form">
        <div class="contacts__inputs">
          <input type="text" placeholder="Your name*">
          <input type="email" name="email" placeholder="Your e-mail*" size="20">
          <input type="text" placeholder="Subject">
        </div>
        <div class="contacts__textarea">
          <textarea placeholder="Write Message*"></textarea>
        </div>
      </form>
      <a href="#" class="btn">Send message</a>
  </section>
  <footer class="footer">
    <div class="container">
      <div class="footer__items">
        <a href="#" class="footer__copywrite">© Copyright 2014 FreebiesXpress.com</a>
        <div class="footer__links">
          <a href="#" class="footer__link"><i class="fab fa-twitter"></i></a>
          <a href="#" class="footer__link"><i class="fab fa-facebook-f"></i></a>
          <a href="#" class="footer__link"><i class="fab fa-pinterest-p"></i></a>
          <a href="#" class="footer__link"><i class="fab fa-google-plus-g"></i></a>
        </div>
      </div>  
    </div>
  </footer>
  <script>
    let menuBtn = document.querySelector('.nav__btn');
    let menu = document.querySelector('.nav');
    menuBtn.addEventListener('click', function(){
      menuBtn.classList.toggle('active');
      menu.classList.toggle('active');
    })
  </script>  
</body>
</html>